engine/setup.py

```diff
@@ -82,11 +83,12 @@
     setup_requires=["cython"],
     ext_modules=cythonize(extensions),
     packages=find_packages(include=["bsql_engine", "bsql_engine.*"]),
+    # TODO: force comment to pass style issue
     package_data={
-        "bsql_engine.io": ["bsql_engine/io/cio.pxd"],
-        "bsql_engine.io": ["*.pxd"],
+        # "bsql_engine.io": ["bsql_engine/io/cio.pxd"],
```

Comment:
why was this commented out? do we need it anymore? if not, we should just delete it.

Reply:
removed